### ISSUE TITLE
can't use m1.small for 64bit ami - although amazon allows it

### ISSUE THREAD

**Issue by jbheman** [jtriley/StarCluster#88]
$ starcluster start c 
StarCluster - (http://web.mit.edu/starcluster) (v. 0.9999)
Software Tools for Academics and Researchers (STAR)
Please submit bug reports to starcluster@mit.edu

> > > Using default cluster template: BTcluster
> > > Validating cluster template settings...
> > > !!! ERROR - Cluster settings are not valid:
> > > !!! ERROR - Incompatible node_image_id and node_instance_type:
> > > !!! ERROR - Instance type m1.small is for an i386 platform while
> > > !!! ERROR - ami-969448ff is an x86_64 platform

must be hard-coded?

**Comment by jtriley** [comment]
@jbheman This was fixed yesterday and included in the latest stable 0.93.2 release. It appears you're using a git checkout of StarCluster (indicated by version 0.9999). Could you try pulling the latest changes and see if that helps? Here's my quick test using the latest github code (which currently matches the latest stable release):

```
% starcluster start -s 1 -n ami-999d49f0 -i m1.small test -v
StarCluster - (http://web.mit.edu/starcluster) (v. 0.9999)
Software Tools for Academics and Researchers (STAR)
Please submit bug reports to starcluster@mit.edu

>>> Using default cluster template: smallcluster
>>> Validating cluster template settings...
>>> Cluster template settings are valid
```

**Comment by jbheman** [comment]
stellar!

thanks, j
